src/DataTable/components/Pagination.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+// Actions
+import { changePage } from '../slices/dataTable'
+
+const Pagination = () => {
```

Comment:
I think you can have generic Pagination component wrapped into slice behavior, but that's ok how it is currently.